cglib/src/test/java/net/sf/cglib/proxy/TestEnhancer.java

```diff
@@ -1418,6 +1419,161 @@ public Class findClass(String name) throws ClassNotFoundException {
     assertEquals(Arrays.asList(Object.class, String.class), paramTypes);
   }
 
+    public void testInterfaceBridge() throws Exception {
+        // recent versions of javac will generate a synthetic default bridge for f(Object) in B
+
+        // interface A<T> {
+        //     void f(T t);
+        // }
+        // interface B<T extends Number> extends A<T>{
+        //     void f(T t);
+        // }
+
+        // The test relies on Java 8 bytecode for default methods.
+        Assume.assumeTrue(getMajor() >= 8);
```

Comment:
this doesn't seem to have the desired effect (i'm assuming you wanted this to bail out w/o errors if running on 1.7).. instead it fails on 1.7 with:
  [ERROR] testInterfaceBridge(net.sf.cglib.proxy.TestEnhancer)  Time elapsed: 0 s  <<< ERROR!
org.junit.AssumptionViolatedException: got: <false>, expected: is <true>
	at net.sf.cglib.proxy.TestEnhancer.testInterfaceBridge(TestEnhancer.java:1433)

Reply:
Fixed, thanks.